packages/roosterjs-editor-api/lib/format/setOrderedListNumbering.ts

```diff
@@ -0,0 +1,16 @@
+import splitList from '../utils/splitList';
+import { IEditor } from 'roosterjs-editor-types';
+
+/**
+ * Resets Ordered List Numbering back to 1
+ * @param editor The editor instance
+ * @param list The List that is going to be split
+ * @param startNumber The number of that the splitted list should start
+ */
+export default function setOrderedListNumbering(
+    editor: IEditor,
+    list: HTMLOListElement,
+    startNumber?: number
+) {
+    splitList(editor, list, startNumber);
```

Comment:
Unless this splitList() function is reused by other functions, we don't need to have a separate util file. Just put your code here is ok.